.github/workflows/integration.yml

```diff
@@ -0,0 +1,55 @@
+name: Integration tests
+
+on: [push, pull_request]
+
+jobs:
+  Integration:
+    runs-on: ubuntu-latest
+    if: github.event_name != 'push' || github.repository == 'DIRACGrid/DIRAC'
+    timeout-minutes: 30
+
+    strategy:
+      fail-fast: False
+      matrix:
```

Comment:
At the moment I run the integration tests with a matrix of MySQL and elasticsearch versions but [MySQL 8.0 crashes during install](https://github.com/chrisburr/DIRAC/runs/231706281) and [ES 7.3.2 hangs indefinitely during testing](https://github.com/chrisburr/DIRAC/runs/231706271).

What should I include in the matrix here?

Reply:
We are not prepared for ES 7 (I can't recall why, atm), remove it from your test.

Can we use SLC6 and CentOS7 in the matrix?

Reply:
MySQL crashes as the client in DIRACOS is outdated, [5.7.23 or later is required](https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-23.html#mysqld-5-7-23-security).

> Can we use SLC6 and CentOS7 in the matrix?

Done